bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewersCommand.java

```diff
@@ -124,24 +129,19 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
         }
 
         var updatedLimits = ReviewersTracker.updatedRoleLimits(censusInstance.configuration(), numReviewers, role);
-        if (updatedLimits.get(role) > numReviewers) {
-            showHelp(reply);
-            reply.println("Number of required reviewers of role " + role + " cannot be decreased below " + updatedLimits.get(role));
-            return;
-        }
 
         reply.println(ReviewersTracker.setReviewersMarker(numReviewers, role));
         var totalRequired = updatedLimits.values().stream().mapToInt(Integer::intValue).sum();
-        reply.print("The number of required reviews for this PR is now set to " + totalRequired);
+        reply.print("The total number of required reviews for this PR, " +
+                "which is combined by the configuration and the command, is now set to " + totalRequired);
```

Comment:
I would suggest:

"The total number of required reviews for this PR (including the jcheck configuration and the last /reviewers command) is now set to"

Reply:
Fixed